src/scene/text-bitmap/BitmapText.ts

```diff
@@ -144,4 +145,21 @@ export class BitmapText extends AbstractText<TextStyle, TextStyleOptions> implem
         bounds.minY = (-anchor._y * (height + offset));
         bounds.maxY = bounds.minY + height;
     }
+
+    /**
+     * The resolution / device pixel ratio of the canvas.
+     * @default 1
+     */
+    override set resolution(_value: number)
+    {
+        warn(
```

Comment:
I forget, are we excluding stuff from prod builds? If so, I would only include this in development.

Reply:
nice catch! its now removed for prod